addons/hr_recruitment_survey/models/survey_survey.py

```diff
@@ -14,7 +14,7 @@ def _compute_allowed_survey_types(self):
         super()._compute_allowed_survey_types()
```

Comment:
Commit message should ideally start with a verb, maybe "support i18n for radio_selection_with_filter labels" ?

(Also a few typos "humain" -> "human", "this allow" -> "this allows").